Backend/Backend/Middlewares/ValidateUser.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Backend.Models;
+using Backend.Services;
+using Backend.Services.Interfaces;
+
+namespace Backend.Middlewares;
```

Comment:
rather than middleware, just call this a helper. We already have a folder set up in the backend in the infrastructure folder (although we might want to rename that in the future)

Reply:
I disagree. Because now we can use the middleware on other routes as well to ensure that user is not null. Rather than duplicating the helper each time